akka-docs/src/main/paradox/distributed-data.md

```diff
@@ -157,15 +157,15 @@ Java
 
 For the full documentation of this feature and for new projects see @ref:[Distributed Data - Subscribe](typed/distributed-data.md#subscribe).
 
-You may also register interest in change notifications by sending `Replicator.Subscribe`
-message to the `Replicator`. It will send `Replicator.Changed` messages to the registered
+You may also register interest in change notifications by sending @apidoc[Replicator.Subscribe](akka.cluster.ddata.Replicator.Subscribe)
+message to the `Replicator`. It will send @apidoc[Replicator.Changed](akka.cluster.ddata.Replicator.Changed) messages to the registered
 subscriber when the data for the subscribed key is updated. Subscribers will be notified
 periodically with the configured `notify-subscribers-interval`, and it is also possible to
 send an explicit `Replicator.FlushChanges` message to the `Replicator` to notify the subscribers
```

Comment:
`@apidoc[Replicator.FlushChanges](akka.cluster.ddata.Replicator.FlushChanges$) ` - doesn't find the class. Because `FlushChanges` in `object` in the class `Replicator`?
<img width="1355" alt="Снимок экрана 2021-12-25 в 20 09 16" src="https://user-images.githubusercontent.com/4740207/147390038-1dd44ec8-9606-433a-82ae-69429cf9a932.png">

Reply:
I don't see an obvious way either, let's leave it